example/android/app/src/main/java/com/example/MainApplication.java

```diff
@@ -28,6 +30,8 @@ protected List<ReactPackage> getPackages() {
           List<ReactPackage> packages = new PackageList(this).getPackages();
           // Packages that cannot be autolinked yet can be added manually here, for example:
           // packages.add(new MyReactNativePackage());
+          packages.add(new SketchCanvasPackage());
+          packages.add(new RNCameraPackage());
```

Comment:
Doesn't autolinking do this now? Haven't seen editing this file in years.

Reply:
yeah it's because it's not a direct dependency of the `example/` folder it's a dev dependency of the main module